packages/amplify_datastore/example/lib/models/ModelProvider.dart

```diff
@@ -68,8 +74,10 @@ export 'IntListTypeModel.dart';
 export 'IntTypeModel.dart';
 export 'JSONListTypeModel.dart';
 export 'JSONTypeModel.dart';
+export 'ModelNestedCustomType.dart';
 export 'MultiTypeModel.dart';
 export 'Post.dart';
+export 'SimpleCustomType.dart';
```

Comment:
I think this is okay for now, but I think there is a limit of 33 model types in order for cloud sync to work, and we are either near or at that limit.

Reply:
We could probably combine some of the types that I created for integ tests. I created a model for each type just to make the tests a little bit more independent, but it isn't 100% necessary. We could also just split these into multiple apps at some point, which I think we want to do eventually anyway to test different configs.